packages/graphql-hooks/index.d.ts

```diff
@@ -56,7 +62,7 @@ interface ClientOptions {
   fetch?(url: string, options?: object): Promise<object>
   fetchOptions?: object
   logErrors?: boolean
-  onError?({ result, operation }: { operation: Operation, result: Result }): void
+  onError?({ operation: Operation, result: Result }): void
```

Comment:
this is actually a regression of #175
please revert it.

```suggestion
  onError?({ result, operation }: { operation: Operation, result: Result }): void
```

Reply:
Oops! I think I had pasted this in from an out of date version. Prettier changed the formatting so I missed the diff.